### ISSUE TITLE
`if: Symbol’s value as variable is void: native-comp-driver-options`

### ISSUE THREAD

**Issue by egh** [alexmurray/emacs-snap#39]
OS: Ubuntu 20.04
emacs: snap
```
emacs                    29.0.50-master-825b543      1553   latest/edge      alexmurray         classic
```

`emacs -Q` loads fine. `emacs -q` causes the following error:
```
if: Symbol’s value as variable is void: native-comp-driver-options
```

Unfortunately, `emacs -q --debug-init` doesn't provide additional information.

**Comment by alexmurray** [comment]
bah apologies for this - I have a fix in progress which I believe should resolve this - it is currently building so should be available in the edge channel in ~1-2 hours. Thanks for reporting this issue so quickly.

**Comment by alexmurray** [comment]
Ok updated snap build are available in the beta channel (for emacs 28.1) and the edge channel (for emacs 29.0.50) for amd64 at least - please let me know if these resolve the issue for you.

**Comment by egh** [comment]
Fixed for me, thanks!
